src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -8681,6 +8697,25 @@ public void PathMapParser()
             Assert.Equal((int)ErrorCode.ERR_InvalidPathMap, parsedArgs.Errors[0].Code);
         }
 
+        [WorkItem(7588, "https://github.com/dotnet/roslyn/issues/7588")]
+        [Fact]
+        public void Version()
```

Comment:
The expectations will be set whether we want this to happen or not. Changing behavior in the future will be a breaking change, therefore, we should have regression tests to prevent the accidental change in behavior. In addition, code reviewers won't have guess what the current behavior is in order to make a decision whether it meets their expectations or not.

---
In reply to: [80583354](https://github.com/dotnet/roslyn/pull/13955#discussion_r80583354) [](ancestors = 80583354,80535339,80532729)